toolbox/_common.py

```diff
@@ -107,4 +106,6 @@ def run_ansible_playbook(playbook_name, opts: dict = dict()):
         for k, v in os.environ.items():
             print(f"{k}={v}", file=f)
 
-    raise SystemExit(run_result.returncode)
+    os.execvpe(cmd[0], cmd, os.environ)
```

Comment:
Isn't this already a part of another pull request?

Reply:
this commit was part of https://github.com/openshift-psap/ci-artifacts/pull/235 but I moved it here, along with the "various updates" of the PR title